jest.config.js

```diff
@@ -1,5 +1,10 @@
 'use strict'
 
+// eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
https://github.com/mojaloop/auth-service/pull/29/files#diff-e4403a877d80de653400d88d85e4801aR59
this rule negates the need for this yes?